### DISCUSSION TITLE
Neeo

### DISCUSSION THREAD

**Discussion by ratkins1962** [community/community#41632] [Other Feature Feedback, Questions, & Ideas]
Hi guys 
New here not sure if anyone can help. I have a Neeo remote and support is now finished, i need to change my tv, amp etc on the system. I thought i saw something on here to do with this?
Does anyone know of a software i can use ?
Any help appreciated

**Comment by queenofcorgis** [comment]
Hi @ratkins1962,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this product, such as  https://www.neeo.com/. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
